RecoJets/JetAnalyzers/src/myFilter.cc

```diff
@@ -804,9 +787,7 @@ myFilter::filter(edm::Event& evt, edm::EventSetup const& es) {
 	//	std::cout << *j << std::endl;
       }
     }
-  } catch (...) {
-    cout << "No HO RecHits." << endl;
-  }
+   ****/
```

Comment:
I commented it out because the only thing it did was loop. There was a cout stamement that this loop could have engaged, but that was already commented out in the code as I found it.  The code was effectively:

for(i in collection){
 // cout<<"we have the collection"<<endl;
}

I went the step further to say that there is no point in looping.